libraries/wallet/wallet.cpp

```diff
@@ -770,6 +770,15 @@ class wallet_api_impl
 
       return true;
    }
+
+    void quit()
+    {
+        ilog( "exiting wallet ..." );
+        save_wallet_file();
+        ilog( "closed wallet successfully" );
+        std::exit(0);
```

Comment:
I don't think it's good to call `exit` in a library.

Reply:
what is the best to call instead of exit in your opinion ? @abitmore can you help please with right direction ?

Reply:
@abitmore sorry maybe I missunderstood you about exit in a library, but here in cli_wallet it's OK do you think ? right ? :-)